llvm/test/Other/dump-before-after.ll

```diff
@@ -0,0 +1,56 @@
+; RUN: rm -rf %t/logs
+
+; Basic dump before and after a single module pass
+
+; RUN: opt %s -disable-output -passes='no-op-module' -ir-dump-directory %t/logs -print-after=no-op-module -print-before=no-op-module
+; RUN: find %t/logs -type f -printf '%P\n' | sort -n | FileCheck %s --check-prefix=SINGLE-PASS
+; SINGLE-PASS: {{[0-9]+}}-[[MODULE_NAME_HASH:[a-z0-9]+]]-module-NoOpModulePass-after.ll
```

Comment:
add some simple checks for the content of one of these

Reply:
Added